src/patternfly/components/LabelGroup/examples/LabelGroup.md

```diff
@@ -554,7 +554,148 @@ In addition to the JavaScript management of [editable labels](/components/label#
+      {{#> label-group-list-item}}
+        {{#> label label--modifier="pf-m-green"}}
+          {{#> label-icon}}
+            <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+          {{/label-icon}}
+          Label 3
+        {{/label}}
+      {{/label-group-list-item}}
+    {{/label-group-list}}
+  {{/label-group-main}}
+{{/label-group}}
+```
+
+### Compact labels, overflow
+```hbs
+{{#> label-group label-group--id="label-group-overflow"}}
+  {{#> label-group-main}}
+    {{#> label-group-list label-group-list--attribute='aria-label="Group of labels"'}}
+      {{#> label-group-list-item}}
+        {{#> label label--modifier="pf-m-compact"}}
+          {{#> label-icon}}
+            <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+          {{/label-icon}}
+          Label
+        {{/label}}
+      {{/label-group-list-item}}
+      {{#> label-group-list-item}}
+        {{#> label label--modifier="pf-m-compact pf-m-blue"}}
+          {{#> label-icon}}
+            <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+          {{/label-icon}}
+          Label 2
+        {{/label}}
+      {{/label-group-list-item}}
+      {{#> label-group-list-item}}
+        {{#> label label--modifier="pf-m-compact pf-m-green"}}
+          {{#> label-icon}}
+            <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+          {{/label-icon}}
+          Label 3
+        {{/label}}
+      {{/label-group-list-item}}
+      {{#> label-group-list-item}}
+        {{#> label label--IsOverflow="true" label--modifier="pf-m-compact"}}
+          3 more
+        {{/label}}
+      {{/label-group-list-item}}
+    {{/label-group-list}}
+  {{/label-group-main}}
+{{/label-group}}
+```
+
+### Compact labels, vertical
+```hbs
+{{#> label-group label-group--id="label-group-vertical" label-group--modifier="pf-m-vertical"}}
+  {{#> label-group-main}}
+    {{#> label-group-list label-group-list--attribute='aria-label="Group of labels"'}}
+      {{#> label-group-list-item}}
+        {{#> label label--modifier="pf-m-compact"}}
+          {{#> label-icon}}
+            <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+          {{/label-icon}}
+          Label
+        {{/label}}
+      {{/label-group-list-item}}
+      {{#> label-group-list-item}}
+        {{#> label label--modifier="pf-m-compact pf-m-blue"}}
+          {{#> label-icon}}
+            <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+          {{/label-icon}}
+          Label 2
+        {{/label}}
+      {{/label-group-list-item}}
+      {{#> label-group-list-item}}
+        {{#> label label--modifier="pf-m-compact pf-m-green"}}
+          {{#> label-icon}}
+            <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+          {{/label-icon}}
+          Label 3
+        {{/label}}
+      {{/label-group-list-item}}
+    {{/label-group-list}}
+  {{/label-group-main}}
+{{/label-group}}
+```
+
+## Accessibility
```

Comment:
Nice catch! We're actually missing a `## Documentation` heading here, and the existing a11y/usage headings should remain `h3`'s - do you mind updating that? Here's an example - https://www.patternfly.org/v4/components/alert/html -- https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Alert/examples/Alert.md

Reply:
oh yes, that makes much sense. i'll update it to match